frontend/src/components/Store.tsx

```diff
@@ -1,23 +1,82 @@
-import React from "react";
-import { StoreGameState } from "../models/game-state.model";
+import React, { useState } from "react";
+import { StoreGameState, UnitPlacement } from "../models/game-state.model";
 import StoreUnit from "./StoreUnit";
+import Unit from "./Unit";
+import { Container, Row, Col } from "reactstrap";
+import { BOARD_HEIGHT, BOARD_WIDTH } from "../constants";
 
 interface Props extends StoreGameState {
   buyUnit: (unitId: string) => void;
+  placeUnit: (unitsPlacement: UnitPlacement) => void;
 }
 
-const Store: React.FunctionComponent<Props> = ({ store, buyUnit }: Props) => {
+const Store: React.FunctionComponent<Props> = ({
+  store,
+  units,
+  unitsPlacement,
+  buyUnit,
+  placeUnit,
+}: Props) => {
+  const range = (n: number): number[] => {
+    return Array.from(Array(n).keys());
+  };
+  const [selectedUnit, selectUnit] = useState<string>("");
+  const board = range(BOARD_HEIGHT).map(_ => range(BOARD_WIDTH).map(_ => ""));
```

Comment:
There should be a separate component for board, it shall not be placed in store. But I'll take that over.

Reply:
Good point!